v1/src/main/proto/export.proto

```diff
@@ -57,4 +57,10 @@ message Export {
   repeated Table change_streams = 3;
   // The set of sequences that were exported.
   repeated Table sequences = 5;
+
+  // Proto descriptors associated with the database
+  bytes proto_descriptors = 6;
+
+  // Proto Bundle associated with the database
+  repeated string proto_bundle = 7;
```

Comment:
Should this be called "proto_fqn" or "proto_types"? Bundle seems to convey a combination of descriptors+types data.

Reply:
In terms of schema export template, this represents the proto_bundle object of the schema similar to Tables/ChangeStreams/Sequences objects above.